### PR TITLE
Pt 01/feature add routes

### PR DESCRIPTION
**Pull request by Agne-stat**
- routes
- connecting database
- create, delete logic
- minor styling

### CODE REVIEW

File: app/components/Calendor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import moment from 'moment'
+
+
+export default function Calendor() {
+    let now = moment().calendar(); 
+           
+  return (
+    <div>
+      <h2>Calendor</h2>
```
Comment (hitunes) [inline app/components/Calendor/index.tsx]: Replace all Calendor with Calendar

File: app/routes/dayPlan.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { LinksFunction, LoaderFunction } from "@remix-run/node";
+import { Outlet, useLoaderData, Link } from "@remix-run/react";
+import { db } from "~/utils/db.server";
+import { json } from "@remix-run/node";
```
Comment (hitunes) [inline app/routes/dayPlan.tsx]: you can merge this with line 1

File: app/routes/dayPlan/$dayPlanId.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import { ActionFunction, LoaderFunction, redirect } from "@remix-run/node";
+import { json } from "@remix-run/node";
```
Comment (hitunes) [inline app/routes/dayPlan/$dayPlanId.tsx]: same, merge with line 1

File: app/routes/dayPlan/$dayPlanId.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    throw new Response(
+      `The _method ${form.get("_method")} is not supported`,
+      { status: 400 }
+    );
+  }
+  const dayPlan = await db.dayPlan.findUnique({
+    where: { id: params.dayPlanId },
+  });
+  if (!dayPlan) {
+    throw new Response("Can't delete what does not exist", {
```
Comment (hitunes) [inline app/routes/dayPlan/$dayPlanId.tsx]: for security reasons this kinda message is bad... we will discuss it later

File: app/routes/index.tsx [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+  );
 }
+
+
+
+
+
+
+
+
```
Comment (hitunes) [inline app/routes/index.tsx]: why these empty lines?

File: app/routes/todo.tsx [hunk 6]
```diff
@@ -1,12 +1,44 @@
...
+import { LinksFunction, LoaderFunction } from "@remix-run/node";
+import { Outlet, useLoaderData, Link } from "@remix-run/react";
+import { db } from "~/utils/db.server";
+import { json } from "@remix-run/node";
+
+import stylesUrl from "~/styles/index.css";
+
+export const links: LinksFunction = () => {
+  return [{ rel: "stylesheet", href: stylesUrl }];
+};
```
Comment (hitunes) [inline app/routes/todo.tsx]: this function is not used here... maybe it doesn't belong here

File: app/routes/todo.tsx [hunk 7]
```diff
@@ -1,12 +1,44 @@
...
 export default function TodoRoute() {
+  const data = useLoaderData<LoaderData>();
   return (
-    <div>
-      <h1>Todo View</h1>
       <main>
         <Outlet />
+        <div>
+          <ul>
+            {data.todoListItems.map((todo) => (
```
Comment (hitunes) [inline app/routes/todo.tsx]: destructure {id, name}

File: app/routes/todo/$todoId.tsx [hunk 8]
```diff
@@ -1,9 +1,95 @@
+import { ActionFunction, LoaderFunction, redirect } from "@remix-run/node";
+import { json } from "@remix-run/node";
```
Comment (hitunes) [inline app/routes/todo/$todoId.tsx]: merge with line 1

File: app/routes/todo.tsx (hunk 6 above)
Comment (Agne-stat) [reply app/routes/todo.tsx]: This is used for styling

Comment (hitunes) [reply app/routes/todo.tsx]: Used for styling in this file or another file?

Comment (Agne-stat) [reply app/routes/todo.tsx]: For all route todo.tsx, dayPlan.tsx and expensesPlan.tsx files

**Pull request closed without merging**